transcriptic/config.py

```diff
@@ -218,12 +218,14 @@ def api_root(self, value):
     @property
     def organization_id(self):
         try:
-            return self.env_args["org_id"]
+            return self.session.headers["X-Organization-Id"]
         except (NameError, KeyError):
             raise ValueError("organization_id is not set.")
 
     @organization_id.setter
     def organization_id(self, value):
+        self.update_headers(**{"X-Organization-Id": value})
+        self.update_session_auth()
```

Comment:
Is this required? I think `update_session_auth()` is called in other instances (e.g. in the `email` setter) because it uses those properties internally:

```
    def update_session_auth(self, use_signature=True):
        if (
            use_signature
            and self._rsa_secret
            and "X-User-Email" in self.session.headers
        ):
            self.session.auth = StrateosSign(
                self.email, self._rsa_secret, self.api_root
            )
        elif self.bearer_token:
            self.session.auth = StrateosBearerAuth(self.bearer_token, self.api_root)
        else:
            self.session.auth = None
```

But `organization_id` is not used internally by `update_session_auth()`, so I don't _think_ it should be necessary.